crates/xilem_core/src/view/adapt.rs

```diff
@@ -189,13 +175,10 @@ macro_rules! generate_adapt_state_view {
             }
         }
 
-        impl<
-                ParentT,
-                ChildT,
-                A,
-                V: $viewtrait<ChildT, A>,
-                F: Fn(&mut ParentT) -> &mut ChildT + Send,
-            > $viewtrait<ParentT, A> for AdaptState<ParentT, ChildT, V, F>
```

Comment:
This is what I mean about the rustfmt output, for some reason it was previously indenting generics too much.